line-bot-spring-boot/src/main/java/com/linecorp/bot/spring/boot/support/LineMessageHandlerSupport.java

```diff
@@ -75,8 +78,8 @@
 @Import(ReplyByReturnValueConsumer.Factory.class)
 @ConditionalOnProperty(name = "line.bot.handler.enabled", havingValue = "true", matchIfMissing = true)
 public class LineMessageHandlerSupport {
-    private static final Comparator<HandlerMethod> HANDLER_METHOD_PRIORITY_COMPARATOR =
-            Comparator.comparing(HandlerMethod::getPriority).reversed();
+    private static final Ordering<HandlerMethod> HANDLER_METHOD_PRIORITY_COMPARATOR =
+            Ordering.natural().onResultOf(HandlerMethod::getPriority).reverse();
```

Comment:
It seems to have been overwritten because I worked with a 1.6.0 release-based source.
This part can be ignored.